src/app/components/popup-choose-the-time/popup-choose-the-time.component.html

```diff
@@ -1,19 +1,26 @@
 <app-modal-window [title]="title">
   <div class="wrapper-content">
     <div class="wrapper-data" [title]="titleData">
+      <span class="title-data">{{ titleData }}</span>
```

Comment:
pls create and use translate service, if it doesn't datatime

Reply:
add